state-network.md

```diff
@@ -119,36 +119,38 @@ Examples:
 
 ##### Merkle Patricia Trie (MPT) Proofs
 
-Merkle Patricia Trie (MPT) proofs consist of a list of `WitnessNode` objects that correspond to individual trie nodes from the MPT. Each node can be one of the different node types from the MPT (e.g.blank, branch, extension, leaf).  When serialized, each `WitnessNode` is represented as an RLP serialized list of the component elements. The largest possible node type is the branch node which when serialized is a list of up to sixteen hashes in `Bytes32` (representing the hashes of each of the 16 nodes in that branch and level of the tree) plus the 4 elements of the node's value (balance, nonce, codehash, storageroot) represented as `Bytes32`.  When combined with the RLP prefixes, this yields a possible maximum length of 667 bytes.  We specify 1024 as the maximum length due to constraints in the SSZ spec for list lengths being a power of 2 (for easier merkleization.)
+Merkle Patricia Trie (MPT) proofs consist of a list of `TrieNode` objects that correspond to
+individual trie nodes from the MPT. Each node can be one of the different node types from the MPT
+(e.g. blank, branch, extension, leaf).  When serialized, each `TrieNode` is represented as an RLP
+serialized list of the component elements. The largest possible node type is the branch node, which
+can be up to 532 bytes (16 child nodes of `Bytes32` with extra encoding info). We specify 1024 as
+the maximum length due to constraints in the SSZ spec for list lengths being a power of 2 (for
+easier merkleization).
 
```

Comment:
This is not really correct. You can still define 532 as max. When merkleized, chunks will get zeropadded to size 32 and there will be zeroed leaves.

Reply:
I'm not familiar with the details of the ssz and merkleization, so I left it as it was. I'm fine with removing those comments.

Although I'm quite certain that 532 bytes is the maximum (also based on observed data), I worried that I might have missed something so I'm in favor of having bigger size. In that regard, 1024 still sounds good to me. Thoughts?

Reply:
Now that I think about it, if chucks are padded to size 32, does it make a difference if we set max to 532 or 1024?

Reply:
De-facto size limit would be 1024, since it hash to be power of 2. So we might as well set it to 1024. Or better yet, say it should be 532 but de-facto limitation is 1024.